scss/_commercial.scss

```
@@ -0,0 +1,178 @@
+.commercial-navbar {
+  background-color: $orange;
+  margin-top: 0px;
+  .navbar-logo {
+    height: 80%;
+    margin: auto 10px;
+  }
+}
+.commSwiper {
+  width: 100%;
+  height: 100%;
+  margin-top: 100px;
+
+  .swiper-slide {
+    text-align: center;
+    font-size: 18px;
+    background: #fff;
+    transition: filter 1s;
+
+    &-prev {
+      filter: brightness(80%);
+      transition: filter 1s;
+    }
+
+    &-active {
+      & img {
```

Ne treba ti & da bi nestovao. & bi trebao da koristis samo za pseude selektore, BEM ili modifier selectore za isti element

```suggestion
      img {
```